torch/_inductor/decomposition.py

```diff
@@ -56,6 +56,9 @@
 )
 decompositions = {**core_aten_decompositions(), **inductor_decompositions}
 
+# Remove unwanted decompositions included via the core ATen decompositions from
+# the Inductor decomp table.
+decompositions.pop(aten._unsafe_index.Tensor, None)
```

Comment:
Please create a exclude_decomp_list, and iterate over the list to exclude. 
We have have more then one decomp to exclude.